src/components/ArticleMenu.js

```diff
@@ -45,6 +51,7 @@ export const ArticleMenu = ({
   }, [])
 
   const items = [
+    { title: i18n.i18n('search-placeholder'), action: onSearchSelected },
     { title: i18n.i18n('article-action-sections'), action: onTocSelected },
     { title: i18n.i18n('menu-textsize'), action: onTextsizeSelected }
   ]
```

Comment:
Thing come in my mind for the refactoring.

The list is getting bigger, instead of push & unshift to the list, we can use the full list and filter it based on the condition, a lot easier for us to know the order of the item.

Reply:
should this be fixed by @medied on this PR or is just a comment for future work?

Reply:
Future